.github/workflows/feature-branch-image.yml

```diff
@@ -0,0 +1,28 @@
+---
+name: Build and Push Feature Branch Custom Catalog
+env:
+  BUNDLE_IMG: quay.io/${{ secrets.QUAY_ORG }}/opendatahub-operator:${GITHUB_REF##*/}
```

Comment:
isnt bundle in quay.io/${{ secrets.QUAY_ORG }}/opendatahub-operator-bundle:${GITHUB_REF##*/}